### PR TITLE
Rename all references to kpod to podman

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @TomSweeneyRedHat @baude @umohnani8 PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.  Hard to tell if any were missed.  I say press the merge button while looking between your fingers.  I'd also suggest holding the merge to Monday morn.  Murphy lets the chances go up exponentially for problems from PR's like this on a Friday night.

File: vendor/github.com/kubernetes-incubator/cri-o/README.md [hunk 1]
```diff
@@ -40,38 +40,38 @@ It is currently in active development in the Kubernetes community through the [d
...
-| [kpod-stop(1)](/docs/kpod-stop.1.md)                 | Stops one or more running containers                                      ||
-| [kpod-tag(1)](/docs/kpod-tag.1.md)                   | Add an additional name to a local image                                   |[![...](/docs/play.png)](https://asciinema.org/a/133803)|
-| [kpod-umount(1)](/docs/kpod-umount.1.md)             | Unmount a working container's root filesystem                             ||
-| [kpod-unpause(1)](/docs/kpod-unpause.1.md)           | Unpause one or more running containers                                    |[![...](/docs/play.png)](https://asciinema.org/a/141292)|
-| [kpod-version(1)](/docs/kpod-version.1.md)           | Display the version information                                           |[![...](/docs/play.png)](https://asciinema.org/a/mfrn61pjZT9Fc8L4NbfdSqfgu)|
-| [kpod-wait(1)](/docs/kpod-wait.1.md)                 | Wait on one or more containers to stop and print their exit codes||
+| [podman(1)](/docs/podman.1.md)                           | Simple management tool for pods and images                                ||
```
Comment (mheon) [inline vendor/github.com/kubernetes-incubator/cri-o/README.md]: No need to make changes in vendor/
We should get a PR into CRI-O to fix readme and transfer there and revendor to remove the last few mentions

**Comment by mheon** [comment]
One extra file in vendor got added otherwise LGTM

File: vendor/github.com/kubernetes-incubator/cri-o/README.md (hunk 1 above)
Comment (rhatdan) [reply vendor/github.com/kubernetes-incubator/cri-o/README.md]: oops.
Fixed

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r=umohnani8

**Pull request closed without merging**